content/en/users/aai/check-in/faq/_index.md

```diff
@@ -9,6 +9,19 @@ description: >
   Most frequent questions about EGI Check-in
 ---
 
+## Getting help
+
+Requests for technical assistance and other issues related to Check-in can be
+obtained by sending an e-mail to:
```

Comment:
```suggestion
obtained by sending an email to:
```
To fix https://github.com/EGI-Federation/documentation/actions/runs/4172993713/jobs/7224780967#step:4:5470